### ISSUE TITLE
GoSublime formatting by removing additional spaces and convertering the rest to tabs.

### ISSUE THREAD

**Issue by GraydenH** [DisposaBoy/GoSublime#637]
After saving a Go file (as Go or GoSublime in the bottom right) any spaces seemingly seamed as extraneous are deleted or replaced by tabs. What this looks like is me trying to use spaces instead of tabs (I've customized my sublime settings along with the GoSublime ones in the .sublime-settings files to use spaces not tabs, do 4 spaces a per tab and not check indentation) like this: 

![image](https://cloud.githubusercontent.com/assets/12054056/8667538/014de10e-29b6-11e5-8c19-2acb153909fe.png)

and after saving it is changed to look like this:

![image](https://cloud.githubusercontent.com/assets/12054056/8667556/32147262-29b6-11e5-9b84-acf9f943c35b.png)

Where spaces are dots and tabs are lines. It seems to decide how many spaces to make per tab from the first time you save the file.

It reverted any attempt to indent in a way that it doesn't agree with and replaces all leading spaces with tabs. I uninstalled GoSublime and the problem went away.

I was wondering if I could some how disable it from formatting for me, because I'd rather use GoSublimes color highlighting than the default sublime one. I've tried messing around in GoSublimes .sublime-settings files and nothing I tried worked.

**Comment by GraydenH** [comment]
I found the root of the problem in the gosublime.sublime-settings file. Not the user one.

it has 3 settings like this that control the the gofmt which auto formats after every save.

![image](https://cloud.githubusercontent.com/assets/12054056/8888106/cf2450a8-3254-11e5-98a9-06bc82c218ba.png)

It turns out it comes pre set with some pretty odd setting like fmt_tab_indent being true even though it's universally accepted that spaces are supposed to be used and not tabs, unless something is different about the standard for Go, in which case I would be inclined to call the Go standard wrong.

Anyway. I just searched fmt and found these settings, so if you want to fiddle with gofmt this is where you find it.
